frontend_lib/src/component/Tags/Tag.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { Checkbox } from '../Input/Checkbox.jsx'
+import PropTypes from 'prop-types'
+
+require('./TagList.styl')
+
+export const Tag = props => {
+  return (
+    <div className='taglist__list__item'>
+      <Checkbox
+        onClickCheckbox={props.onClickCheckbox}
+        checked={props.checked}
+        name={'tag-' + props.tagId}
+        styleLabel={{ marginLeft: '5px', marginRight: '10px' }}
+        styleCheck={{ top: '-5px' }}
+      />
+      <label
+        htmlFor={'checkbox-tag-' + props.tagId}
+        className='taglist__list__item__info'
+        title={props.name}
+      >
+        {props.name}
+      </label>
+    </div>
+  )
+}
+
+export default translate()(Tag)
+
+Tag.defaultProps = {
+  checked: false,
+  onClickCheckbox: () => { }
+}
+
+Tag.propTypes = {
+  checked: PropTypes.bool,
+  name: PropTypes.string.isRequired,
+  tagId: PropTypes.number.isRequired,
+  description: PropTypes.string,
+  onClickCheckbox: PropTypes.func
+}
```

Comment:
Missing description in defaultProps.

Note: normally we put propTypes first with the required properties at the beginning and then defaultProps. It is a good practice to make it easier to read and to emphasize the important information, but it is not a rule, you can leave it like this.

Reply:
by the way description should be dropped, we are not using it for now (@gbonaspetti @charline-rageade)

Reply:
done